website/source/docs/enterprise/runs/variables-and-configuration.html.md

```diff
@@ -75,9 +90,44 @@ For any of the `GITHUB_` attributes, the value of the environment variable will
 be the empty string (`""`) if the resource is not connected to GitHub or if the
 resource was created outside of GitHub (like using `terraform push`).
 
+### Personal Variables
+
+Personal variables can be created at the Environment or Organization level. All
+Personal variables are private and scoped to the user that created them. Just
+like shared Environment variables, they are injected into the virtual 
+environment during the `plan` and `apply` phases.
+
+Both Personal Environment and Personal Organization variables can be used to
+override Environment variables on a per-user basis. 
+
+## Variable Hierarchy 
+
+It is possible to create the same variable in multiple places for more granular
+control. Variables are applied in the following order:
+
+1. Environment
```

Comment:
FWIW we did recently internally plan to rename what Terraform and Terraform Enterprise both (separately) call "environment", and this was one of the reasons why... too many overloaded meanings of "environment" in our world.

That change is still under discussion internally so it's too early to update the docs for it yet, but see the internal RFC TFE-029 for more info on the latest thinking about how to resolve these terminology issues.